style.css

```diff
@@ -0,0 +1,89 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;900&display=swap");
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+}
+
+body {
+  box-sizing: border-box;
+  font-family: "Poppins", sans-serif;
+  background-position: 20px 10px;
+  background-image: url(./header-shapes\ mobile.png);
+  background-size: cover;
+  background-position-y: 1px;
+  background-position-x: 30px;
+  background-repeat: no-repeat;
+}
+
+.header-content {
+  justify-content: space-between;
+  width: 100%;
+  align-items: center;
+  position: inherit;
+  margin: 0 0;
+  padding: 0;
+}
+
+.sub-header-container {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo img {
+  height: 40px;
+  width: 40px;
+  margin: 20px;
+  background-color: black;
+  border-radius: 50%;
+  padding: 2px;
+  align-content: center;
+  text-align: center;
+}
+
+.bars-icon {
+  margin: 20px;
+  align-content: center;
+  color: rgb(32, 28, 87);
+}
+
+.image-intro i {
+  margin-right: 10px;
+  font-size: 25px;
+  color: rgb(38, 38, 73);
+}
```

Comment:
- [ ] The alignment of social icons should be horizontally placed, Kindly fix it.
- Original
![new1](https://user-images.githubusercontent.com/26767383/149211263-e2a0f802-ec71-489c-877f-d07737013a0d.PNG)
- your website
![new2](https://user-images.githubusercontent.com/26767383/149211420-a2661e3a-579d-4cf6-99bc-fd1f669cd6bd.PNG)